03/app.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {
+    createRoot
+} from 'react-dom/client';
+import MenuItem from './Components/MenuItem';
+
+const root = createRoot(document.querySelector('#root'));
+const App = () => < MenuItem text = "kontakt"
+url = "/contact" / >
```

Comment:
Jak już piszemy w ten sposób to starajmy się to mieć w jednej linii - będzie czytelniej :)